### PR TITLE
Moving FAQ  Dataset Metadata Errors  to GTN

### PR DESCRIPTION
**Pull request by Beachyesh**
This file is meant to answer the FAQ:  How do I find, adjust, and/or correct metadata?

### CODE REVIEW

**Comment by Beachyesh** [comment]
Thanks a lot @beatrizserrano for all the inputs, this clears up a lot of doubts I had while making this file. Will make these changes.

**Comment by beatrizserrano** [comment]
> @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!

yes, this looks better now :) thanks!

**Comment by Beachyesh** [comment]
> > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> 
> yes, this looks better now :) thanks!

That's great! Any other changes before merging?

**Comment by beatrizserrano** [comment]
> > > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> > 
> > 
> > yes, this looks better now :) thanks!
> 
> That's great! Any other changes before merging?

yes, the suggestions I made in my comment above: clear title, clear steps, structured metadata...

**Comment by Beachyesh** [comment]
> > > > @beatrizserrano Does this file seem more coherent with our goals? Let me know. Thanks!
> > > 
> > > 
> > > yes, this looks better now :) thanks!
> > 
> > 
> > That's great! Any other changes before merging?
> 
> yes, the suggestions I made in my comment above: clear title, clear steps, structured metadata...

Okay, so in this updated file, I have taken a specific issue of missing dataset metadata and given a small solution in a few short steps. Accordingly, I have given it a suitable title (to my best knowledge). I also think that a separate faq file for "tool errors related to metadata" and "changing or updating metadata" would be helpful. I can work on it once my other PRs pass their reviews.

File: faqs/galaxy/dataset_metadata_missing.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Missing or Incomplete Dataset metadata 
+area: datasets
```
Comment (annefou) [inline faqs/galaxy/dataset_metadata_missing.md:3]: I am not sure the title reflects what an end users would look for. I mean you would need to know that you have an issue with the metadata of your dataset.

File: faqs/galaxy/dataset_metadata_missing.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+layout: faq         
+---
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon
+- Tools error when using a previously successfully used specific dataset
+- Tools error with a message that ends with: ``OSError: [Errno 2] No such file or directory.``
+
+Solution:
+Click on the the dataset's {% icon galaxy-pencil %} ✏️ to reach the _Edit Attributes_ forms and click either:
```
Comment (annefou) [inline faqs/galaxy/dataset_metadata_missing.md:14]: The formatting will not appear very nicely here. You either need to add a list (or numbered list) or have a blank line in between.

File: faqs/galaxy/dataset_metadata_missing.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- **Directly reset metadata** 
+  - Find the tab for the metadata you want to change, make the change, and save.
+- **Autodetect metadata**
```
Comment (annefou) [inline faqs/galaxy/dataset_metadata_missing.md:17]: So here maybe end users would like to know if they should first reset metadata and then autodetect metadata, or should they choose which one to do? That would be very nice to clarify it. Many thanks.

**Comment by Beachyesh** [comment]
@annefou 
That was a very valid point to consider, how the user will know if they have an issue with metadata, which is why I have added a small introduction in the beginning and renamed the file to "tool errors relating to metadata". 
Also added a line after the Solution to make the format better. 
Finally, also mentioned that the user has to choose one of the two options from the solution as it applies to their situation.

This has made the whole faq slightly longer. Please let me know if that is okay. Looking forward to your review. Thanks!

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq         
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undectected attributes, which require user action (example: assigning database to newly uploaded data)
```
[the remark this answers is not in the document]
Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: Oops! Fixed the typo.

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+layout: faq         
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undetected attributes, which require user action (example: assigning database to newly uploaded data)
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon.
```
Comment (annefou) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:14]: you could use icons here (see https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/faqs/icons_list.html).

Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: Done.

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: jennaj,
```
Comment (annefou) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:5]: It looks weird here.

Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: My mistake.

Comment (annefou) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: it should not be removed but updated to reflect the effective list of current contributors. I guess you want to be in that list too.

Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: Yes, I'm aware, I was confused as to whose names to be added? Just mine or also someone else's?

Comment (beatrizserrano) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: there's not such thing as mandatory contributors here :) Here's the explanation: jennaj wrote the FAQ initially, so she has to be on the contributors list. You moved it to the GTN, so you also deserve to be there. Just add both names (please check how it's done in the other PRs)

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
```
Comment (beatrizserrano) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:2]: hmm I thinking now that we may want to have the title as an explicit question, what do you think @Beachyesh?

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: [jennaj, beachyesh]
+layout: faq
```
Comment (beatrizserrano) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:6]: same comment as in the other PR with the order here

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+contributors: [jennaj, beachyesh]
+layout: faq         
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
```
Comment (beatrizserrano) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:10]:
```suggestion
 - Tool outputs, which are automatically assigned without user action.
```

File: faqs/galaxy/dataset_tool_erorrs_metadata.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undetected attributes, which require user action (example: assigning database to newly uploaded data)
```
Comment (beatrizserrano) [inline faqs/galaxy/dataset_tool_erorrs_metadata.md:12]:
```suggestion
 - Undetected attributes, which require user action (example: assigning database to newly uploaded data).
```

File: faqs/galaxy/dataset_tool_erorrs_metadata.md (hunk 7 above)
Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md:2]: Okay, so this one has felt a little tricky. At first, I kept the title about missing Metadata, but Anne pointed out how will the user know that the error is about the missing metadata? I thought it was a fair point and hence I made the decision to keep the FAQ title slightly broader. Considering this suggestion and usual title guidelines in mind does "Incorrect or Undetected Datatype Attributes" sound like an appropriate title?

**Comment by beatrizserrano** [comment]
@Beachyesh could you rename the file? errors instead of erorrs :)

File: faqs/galaxy/dataset_tool_erorrs_metadata.md (hunk 7 above)
Comment (annefou) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md:2]: Well, we do not know everything and this is why collaborating together is usually best!

Maybe something like: "How to find and correct tool Errors related to Metadata?"

Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_erorrs_metadata.md]: Yess sounds good!

File: faqs/galaxy/dataset_tool_errors_metadata.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+title: How to find and correct tool errors related to Metadata?
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq    
+contributors: [jennaj, beachyesh]
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action.
+ - Incorrect autodetection of datatypes, which need manual modification
```
Comment (annefou) [inline faqs/galaxy/dataset_tool_errors_metadata.md:11]: There is no "." at the end of this sentence. It is recommended to be consistent.

Comment (annefou) [reply faqs/galaxy/dataset_tool_errors_metadata.md:11]: And I see @beatrizserrano already mentioned it! 
It is looking great and we'll be able to merge it soon! Thanks.

Comment (Beachyesh) [reply faqs/galaxy/dataset_tool_errors_metadata.md:11]: Thanks!

**Review by beatrizserrano — APPROVED** [review]

**Comment by beatrizserrano** [comment]
thank you @Beachyesh!

**Pull request merged**